lifecycles.js file is not generated automatically (#566)
strapi/documentation: docs/developer-docs/latest/development/backend-customization/models.md

- with the [Content-Type Builder in the admin panel](/user-docs/latest/content-types-builder/introduction-to-content-types-builder.md),
- or with [Strapi's interactive CLI `strapi generate`](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-generate) command.

The content-types has the following models files:

> - `schema.json` for the model's [schema](#model-schema) definition. (generated automatically, when creating content-type with either method)

Reviewer: I don't think we need the () text at the end

Reviewer 2: ```suggestion
- `schema.json` for the model's [schema](#model-schema) definition. This file is automatically generated when creating a content-type.
```

> - `lifecycles.js` for [lifecycle hooks](#lifecycle-hooks). (must be created manually)

Reviewer 2: ```suggestion
- `lifecycles.js` for [lifecycle hooks](#lifecycle-hooks). This file must be created manually.
```

> - `lifecycles.js` for [lifecycle hooks](#lifecycle-hooks). (can be generated manually)

Reviewer: ```suggestion
- `lifecycles.js` for [lifecycle hooks](#lifecycle-hooks). (must be created manually)
```
